indexer/README.md

```diff
@@ -1,226 +1,153 @@
 ```
 pip install awscli --upgrade
 aws configure
 ```
 
-### Chalice
-
-Chalice is similar to Flask but is serverless and uses AWS Lambda.
-```
-pip install chalice
-chalice new-project
-```
-When prompted for project name, input `<your-indexer-lambda-application-name>`, (e.g., dss-indigo).
-
-Change the working directory to the newly created folder `<your-indexer-lambda-application-name>` (e.g., dss-indigo) and execute `chalice deploy`. Record the URL returned in the last line of stdout returned by this command - henceforth referred to as `<callback_url>`. This will create an AWS Lambda function called `dss-indigo` which will be updated using `chalice deploy`. Chalice automatically generated a folder `chalicelib/` and the files `rm app.py` and `rm requirements.txt`. Overwrite those by copying `app.py`, `requirements.txt` and `chalicelib/` from this repo and to the dss-indigo folder. Then execute
-
-`pip install -r requirements.txt`
-
-### Config File
-
-`chalicelib/config.json` should contain the keys that you wish to add to the index documents. The structure of the config.json should mimic the metadata json file being looked at.
-
-For example, the following metadata for assay.json:
-```
-{
-  "rna": {
-    "primer": "random"
-  },
-  "seq": {
-    "machine": "Illumina HiSeq 2000",
-    "molecule": "total RNA",
-    "paired_ends": "no",
-    "prep": "TruSeq"
-  },
-  "single_cell": {
-    "method": "mouth pipette"
-  },
-  "sra_experiment": "SRX129997",
-  "sra_run": [
-    "SRR445718"
-  ],
-  "files": [
-    {
-      "name": "SRR445718_1.fastq.gz",
-      "format": ".fastq.gz",
-      "type": "reads",
-      "lane": 1
-    }
-  ]
-}
-```
-and this cell.json
-```
-{
-  "type": "oocyte",
-  "ontology": "CL_0000023",
-  "id": "oocyte #1"
-}
-```
-Could have a config like such:
-```
-{
-  "assay.json": [
-    {
-      "rna": [
-        "primer"
-      ]
-    },
+### 3. Setup Chalice
+
+9. Install Chalice with the following command:
+    ```
+    pip install chalice
+    chalice new-project
+    ```
+9. When prompted for project name, input `<your-indexer-lambda-application-name>`.
+9. Change the working directory to the newly created folder `<your-indexer-Lambda-application-name>`.
+9. Execute `chalice deploy`. **Note:** If you are deploying an AWS API Gateway stage other than the default (dev)
+add the `--stage <stage name>` option.
+9. Record the URL returned in the last line of stdout returned by this command - henceforth referred to as `<azul-indexer-url>`.
+This will create an AWS Lambda function with the name of your application name which will be updated using `chalice deploy`.
+9. Chalice will automatically generate a folder with the name of your application name. The folder will contain the files `app.py` and `requirements.txt`
+and a `.chalice` folder containing a `config.json` file. Overwrite those by copying `app.py`, `requirements.txt` and `chalicelib/` from this repo and the generated folder.
```

Comment:
Is
>  ...from this repo and the generated folder.

supposed to read
 >  ...from this repo ~and~ **to** the generated folder.

Also if it's supposed to be 'to', then exactly which files are replaced by what?

Reply:
It's supposed to be "to". 

The generated folder contains a template app.py file and a template requirements.txt file. You are supposed to replace those files with the more complete app.py and requirements.txt from the repo.